### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Trouffman**
Adding extra infos in readme to help with discovery and address most questions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,14 @@ Network Audio/Video in OBS-Studio using NDI technology
 - **NDI Output** : transmit OBS video and audio to NDI
 - **NDI Filter** (a.k.a NDI Dedicated Output) : transmit a single OBS source or scene audio to NDI
 
+DistroAV supports NDI High Bandwidth (encodes/decodes/sends/receives) and can decodes/receives NDI HX2.
+
+## What is NDI ?
+- NDI is a network protocol that allows low-latency, high-quality Audio-Video stream on local network
+- A growing number of devices integrate NDI (PTZ Camera, Encoder, Decoder, etc)
+
+This it NOT a stream distribution protocol. It is mostly aimed at local network use due to bndwidth requirement
```
Comment (paulpv) [inline README.md:28]: Is "a stream distribution protocol" a specific thing/term?
You mean just a "streaming protocol" like RTMP, RTSP, HLS, SRT, WebRTC, etc?

What do you mean by "It is mostly aimed at local network use due to bandwidth requirements."?
Technically everything uses "local network" :)
And "bandwidth requirement" doesn't really mean much to me with no context.
100Mb? 1Gb? 2.5Gb? 5Gb? 10Gb?
Even NDI High Bandwidth would fall apart on 1Gb with 10 streams.

I am wondering if we can just reuse some official NDI marketing phrases from https://ndi.video/tech/ or https://en.wikipedia.org/wiki/Network_Device_Interface .

File: README.md [hunk 2]
```diff
@@ -19,6 +19,14 @@ Network Audio/Video in OBS-Studio using NDI technology
 - **NDI Output** : transmit OBS video and audio to NDI
 - **NDI Filter** (a.k.a NDI Dedicated Output) : transmit a single OBS source or scene audio to NDI
 
+DistroAV supports NDI High Bandwidth (encodes/decodes/sends/receives) and can decodes/receives NDI HX2.
```
[the remark this answers is not in the document]
Comment (Trouffman) [reply README.md:22]: Sure that is fine :)

File: README.md (hunk 1 above)
Comment (Trouffman) [reply README.md:28]: I am not stubborn on the wording, and using what the official NDI page says is preferred to keep communication easy.

The goal is simple for this PR :
- make it clear what the plugin do / doesn't.
- make clear what NDI is, who owns it.
- present a couple of typical use cases :
Receive AV stream from NDI-enabled devices.
Send AV stream to NDI-enabled decoders or other ONS with DistroAV.

**Comment by Trouffman** [comment]
Closing thr PR for now - suggesting changes in its own branch.

**Pull request closed without merging**